back/config/cable.yml

```diff
@@ -1,5 +1,6 @@
 development:
-  adapter: async
+  adapter: redis
+  url: redis://redis:6379
```

Comment:
This url seems to be the only form that does not error, though I'm not convinced it's correct / working.

Other forms tried:
```Ruby
url: redis://localhost:6379/1  => EADDRNOTAVAIL error
url: redis://localhost:6379     => EADDRNOTAVAIL error
url: reds://something:6379/1 => #<SocketError: getaddrinfo: Name or service not known> error
```